frontend/src/test/resources/scala-java-processorpath/build.sbt

```diff
@@ -0,0 +1,25 @@
+bloopConfigDir in Global := baseDirectory.value / "bloop-config"
```

Comment:
Is this needed? It seems to be failing the tests and the defaults should work correctly